src/app/frontend/deploymentdetail/deploymentdetail.html

```diff
@@ -65,5 +65,20 @@
   </kd-content>
 </kd-content-card>
 
+<kd-content-card>
+  <kd-title>[[Horizontal Pod Autoscalers|Title for the 'Horizontal Pod Autoscalers' card on the Replication Controller detail page]]
```

Comment:
![selection_143](https://cloud.githubusercontent.com/assets/1159999/20391825/58277ff0-acd5-11e6-8e1f-99630ba90c4b.png)

I think that now this should be removed.